src/test/java/com/spencerwi/either/EitherCollectorsTest.java

```diff
@@ -0,0 +1,54 @@
+package com.spencerwi.either;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("EitherCollectors with Java")
+public class EitherCollectorsTest {
+
+    @Test
+    public void checkBiasedCollectors(){
+        Either<String, Integer> either1 = Either.left("left");
+        Either<String, Integer> either2 = Either.right(42);
```

Comment:
I think this test would benefit quite a bit from having multiple of each "side" in the list. That would not only better exercise the collector code, but would also serve as a clearer example of what this collector does.

Maybe have a list that goes `Left, Right, Right, Left, Left`, and then assert that the results of each biased-collector are correct.